l10n_br_sale/models/res_config.py

```diff
@@ -18,12 +17,12 @@ def _get_default_copy_note(self):
     copy_note = fields.Boolean(u'Copiar Observações nos Documentos Fiscais',
                                default=_get_default_copy_note)
 
-    def set_sale_defaults(self, cr, uid, ids, context=None):
-        result = super(SaleConfiguration, self).set_sale_defaults(
-            cr, uid, ids, context)
-        wizard = self.browse(cr, uid, ids, context)[0]
-        ir_values = self.pool.get('ir.values')
-        user = self.pool.get('res.users').browse(cr, uid, uid, context)
-        ir_values.set_default(cr, SUPERUSER_ID, 'sale.order', 'copy_note',
-                              wizard.copy_note, company_id=user.company_id.id)
+    @api.multi
```

Comment:
You can improve this like https://github.com/odoo/odoo/blob/10.0/addons/sale/models/sale_config_settings.py#L110